packages/aila/src/utils/experimentalPatches/mathsQuiz.fixture.ts

```diff
@@ -44,4 +44,15 @@ export const mathsQuizFixture: Quiz = [
       "neither similar nor congruent.",
     ],
   },
+  {
+    question:
+      "Here is an image ![image](http://oaknationalacademy-res.cloudinary.com/image/upload/v1706110974/fukcqeavzcevgjhmm1n4.png) and here is the same image  ![image](http://oaknationalacademy-res.cloudinary.com/image/upload/v1706110974/fukcqeavzcevgjhmm1n4.png)",
```

Comment:
This ensure there are 6 questions in the ficutre, and a question that has `{image} {text} {image}`, which was previously failing to replace